programs/marginfi/tests/misc/operational_state.rs

```diff
@@ -1,10 +1,25 @@
 use fixed_macro::types::I80F48;
 use fixtures::{assert_custom_error, prelude::*};
 use marginfi::prelude::MarginfiError;
-use marginfi_type_crate::types::{BankConfig, BankConfigOpt, BankOperationalState};
+use marginfi_type_crate::{
+    constants::EXP_10_I80F48,
+    types::{BankConfig, BankConfigOpt, BankOperationalState},
+};
 use pretty_assertions::assert_eq;
 use solana_program_test::*;
 
+use fixed::types::I80F48 as RuntimeI80F48;
```

Comment:
You don't need this alias, it will work as is:
```suggestion
use fixed::types::I80F48;
```